### PR TITLE
add human-learn

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by raybellwaves** [comment]
`conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {"clumper[version='>=0.2.5']"}`

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=244493&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=597

File: recipes/human-learn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/human-learn/
+  summary: natural intelligence benchmarks for scikit-learn
+  license: MIT
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/human-learn/meta.yaml:43]: Add comment:

```yaml
# license file manually packaged: https://github.com/koaning/human-learn/pull/42
license_file: LICENSE
...
```

I put in a PR upstream to fix having to manually package the license file (https://github.com/koaning/human-learn/pull/42).

Comment (synapticarbors) [inline recipes/human-learn/meta.yaml:43]: Add metadata:

```
dev_url: https://github.com/koaning/human-learn
doc_url: https://koaning.github.io/human-learn/
description: |
    Contains scikit-learn compatible tools that should make it easier to construct 
    and benchmark rule based systems that are designed by humans.
```

**Comment by synapticarbors** [comment]
Waiting on https://github.com/conda-forge/staged-recipes/pull/13322

**Pull request merged**